js/project-list-item.js

```diff
@@ -36,8 +36,17 @@ var ProjectListItem = {
     var html = ProjectListItem.buildHTML(projectData);
     var $listItem = $(html);
 
+    $listItem.find(".star").on("click", function(event) {
+      event.preventDefault();
+      FavouritesManager.toggleProjectFavState(id,projectData.Title);
+
     $listItem.find(".project-title").on("click", ProjectListItem.viewProjectLinkClickHandler);
 
+      var favedProjects = FavouritesManager.getFavedProjects();
+      favs_count = _.size(favedProjects);
```

Comment:
We should rename `favs_count` in `view-manager.js` to `favsCount` since camelCase is JavaScript naming convention for variables. 

Do you mind changing this one to `favsCount` as well as the one in `view-manager.js`?